wordpress-mobile/WordPress-iOS
# Integrate update to WPAuthenticator that allows out apps to hide the WP.com TOS

Ref: wordpress-mobile/WordPressAuthenticator-iOS#536
Ref: woocommerce/woocommerce-ios#3301

This is a draft PR to test that a change to the API in WordPressAuthenticator introduced in wordpress-mobile/WordPressAuthenticator-iOS#536 does not break anything.

## Testing:
* Checkout the branch. Run `bundle exec pod install`
* Log out if necessary
* Attempt to login with a WordPress.com account
* Notice that the TOS are still visible and functional.

<img src="https://user-images.githubusercontent.com/2722505/101437290-cdea9c80-394a-11eb-9f8b-ac3c34a7b4c9.png" width="350"/>


PR submission checklist:
